### ISSUE TITLE
Your shop is being processed

### ISSUE THREAD

**Issue by NicoleParberry** [facebookincubator/facebook-for-woocommerce#357]
I have added my WooCommerce site to my facebook business page. I am still getting the message "Your shop is being processed
Once processing is complete, your shop will be visible to people who visit your Page. You can still create and run ads promoting your products on Facebook."

It has been over a week. How long does it take to get approved to sell on facebook? Is there something I am missing to get approved? My facebook page is Facebook.com/VioletBreezeBoutique
Thanks!

**Comment by mengyingdu** [comment]
Hi @NicoleParberry ,

I checked our fb page and store, your shop has some failure after review. I confirmed that your pixel has been fired correctly which means you installed plugin correctly. I have already file this issue with our shop team to help triage your problems.

Thank you.

**Comment by NicoleParberry** [comment]
Thank you so much!!!  I really appreciate your help!

**Comment by knightofn1** [comment]
i think i'm having the same problem, mengy could you help me as well? pleaseee

**Comment by mengyingdu** [comment]
Hi @knightofn1 ,

What's your fb page and store url?
Next time when you open an issue, can you follow the instruction under support section of the repository? This can help us troubleshooting your problem even faster. If you only tell us and ask for help, we don't know anything about you...

Thanks.

**Comment by propbono** [comment]
Hi @mengyingdu,

it seems I have the same problem with my FB shop. I have imported my products from Woocommerce, but it is almost a week and my shop is still processing. Can you please take a look.

My FB page: `https://facebook.com/Gemprint_GTA`
Shop Page: `https://facebook.com/Gemprint_GTA/shop`
Website: `http://gemprint.ca/shop`

I will appreciate any help.

Thank you,

Propbono

**Comment by FlairTrade** [comment]
I'm having this same issue, was in this state for a few days this past week. I've deleted the plug-in settings and Re-configured Facebook Setting to force it to reload and it's still in this state. It's never taken this long for the shop to be processed after a forced reset so add me to the bug list.

My FB page: https://www.facebook.com/flairtradeconsign/
Shop Page: https://www.facebook.com/flairtradeconsign/shop/
Website: https://www.flairtradeconsign.com/shop/
Using plug-in Version 1.8.2

**Comment by mengyingdu** [comment]
Hi @NicoleParberry ,

Your shop is alive now.

Thanks.

**Comment by mengyingdu** [comment]
Hi @FlairTrade ,

I checked our record, it shows you have set up a self-serve shop on this page around 27 Feb. To clarify, one page only can have one shop. This is why your new shop cant be under review (because it has already have one)
However, for some reason, you didn't set up the self-serve shop correctly. And if you want to use our plugin to sync your products and catalog, you need to use our integration to set up the shop. 

Is is  Ok to disable the self-serve one? If you know how to do it as you created it before, you can simply disable it and re-config your settings on the plugin side.
If you don't know how to do it, let me know if you want us to delete it since we need your permission before we changed your data.

Thanks.

**Comment by FlairTrade** [comment]
@mengyingdu thank you for researching. The 27 Feb  event was when I was attempting a different bug fix but the shop has been successfully operating with this plug-in since that time so this is a true mystery why it's not now. Anyhow, you have my OK to disable the self-serve one since I'm not 100% sure how. Thank you.

**Comment by concypt** [comment]
I am having same issue, my details are: 

My FB page: https://www.facebook.com/raavi.pk/
Shop Page: https://www.facebook.com/raavi.pk/shop/
Website: https://raavi.pk/all-books/
Plugin Version 1.8.2

**Comment by itswenday** [comment]
@mengyingdu I've been in limbo for weeks now waiting for processing to go through. Is there something I did wrong?

My FB page: https://www.facebook.com/scoutunlimited
Shop Page: https://www.facebook.com/scoutunlimited/shop/
Website: https://www.scoutunlimited.com
Just updated to Plugin Version 1.8.3

**Comment by 00cl0ck** [comment]
@mengyingdu Hey dude, I'm having the same problem on my shop, do you mind if you take a look?
Thanks!
My FB page: https://www.facebook.com/Spy-Gifts-627749864223230/
Shop Page: https://www.facebook.com/Spy-Gifts-627749864223230/shop/
Website: https://buyspygifts.com/
Facebook for WooCommerce Plugin Version 1.8.1

**Comment by dmitridr** [comment]
Hey all,

Thanks everyone for reporting this issue. We've been working all day and finally found the problem and fixed it, (it was a bug in our review system for shops), the fix should be live in a few hours for newly created shops. For existing shops, we're planning to fix those by end of day tomorrow. Will update this thread when we've fixed everything. Thanks for your patience!

**Comment by propbono** [comment]
That's amazing news!
Thank you guys for all the hard work.

Cheers! 
Propbono

Sent from Yahoo Mail on Android 
 
  On Tue, 3 Apr 2018 at 10:41 PM, Dmitri Dranishnikov<notifications@github.com> wrote:   
Hey all,

Thanks everyone for reporting this issue. We've been working all day and finally found the problem and fixed it, (it was a bug in our review system for shops), the fix should be live in a few hours for newly created shops. For existing shops, we're planning to fix those by end of day tomorrow. Will update this thread when we've fixed everything. Thanks for your patience!

**Comment by concypt** [comment]
@dmitridr Thats a great news and a bundle of thanks. Looking forward to get it done.

**Comment by mengyingdu** [comment]
Hi @FlairTrade ,

Your shop is alive now.

**Comment by mengyingdu** [comment]
Hi @concypt ,

Your shop is alive now.

**Comment by mengyingdu** [comment]
Hi @00cl0ck ,

Your shop is alive now.

**Comment by itswenzday** [comment]
I think my shop needs some defibrillators. It's going towards the light.

**Comment by dmitridr** [comment]
@itswenzday we managed to bring your shop back from the brink, but it was a close call. Maybe take it easy with the shop for the next few days, give it some time to recuperate!

**Comment by itswenzday** [comment]
It's alive! I'm afraid to see the insurance bill when it comes. 

Thank You @dmitridr

**Comment by dmitridr** [comment]
No problem!

Happy to say that we flat out fixed the problem for 95% of people with this issue, so with luck everyone's shop should be live now. I checked most of the ones in this thread and they are live.

If you're in the 5% we could not fix, try the following, first wait ~2-3 hours from the time of this post (it may still be in the review system because people are reviewing it). If it's still not working, reset the plugin using `Reconfigure > Advanced Options > Delete` and then complete the setup again, then wait 48 hours, the shop should be processed correctly.

If that still does not work, which is very unlikely, please let us know by opening a new issue. That said, most likely everything is working for you now!

**Comment by dotafeeder** [comment]
Hi @mengyingdu,

The item that uploaded in facebook shop are still under processing. I have imported my products from Shopify, but it is almost a week and my shop is still processing. Can you please take a look.

My FB page: https://business.facebook.com/WANGWANG.HEALTHCARE/?business_id=1771064416534690
Shop Page: https://business.facebook.com/pg/WANGWANG.HEALTHCARE/shop/?business_id=1771064416534690
Website: https://www.wangbuu.com/

Appreciate your help.

Thank you

**Comment by milosmili** [comment]
"Your product is being processed
This usually takes a few minutes. We'll notify you when processing is complete and your product is visible."

Some products is being processed more then 3 days, 2 rejected for unknown reasons -> I appealed and no reply so far

FB page: https://www.facebook.com/Topphonestuff-479032609180608/
Shop Page: https://www.facebook.com/pg/Topphonestuff-479032609180608/shop/
Website: https://topphonestuff.com

Thx.

**Comment by equiette** [comment]
Hi there!
I have a few products that have said "Processing". I am unable to tag the products on our Instagram page because of this - I would love to have that feature back.

They have been this way for weeks - this thread is the first I've seen of others experiencing this and finding a solution. I created my listings organically, meaning they are not imported, which I know everyone on this thread seems to have done. Let me know if I'm posting in the wrong place here. I can never find a way to contact Facebook directly.

FB Page: https://www.facebook.com/Benefab/
Shop Page: https://www.facebook.com/pg/Benefab/shop/?ref=page_internal
Missing: Mesh Sheet, SmartScrim, QuickWraps
Website: benefabproducts.com

**Comment by milosmili** [comment]
Equitte, I'd suggest you re-post those...if u'r on WP make sure plugin is activated and update a product..this will force the product to get updated on your FB Shop page

**Comment by scorpiodevelopment** [comment]
My products have been saying processing for a few days and it's not showing up when I'm viewing incognito.  My FB page is https://www.facebook.com/pg/meowsquishies/shop/

When I'm logged in I can see them processing and I can even create categories and sort them.

**Comment by yassersyndicate** [comment]
hello 
it seems I have a problem with my FB shop. I have imported my products from Woocommerce, but it is almost more than  an  2 hour and my shop is still processing. Can you please take a look.

My FB page: https://www.facebook.com/DIYBFYK
Shop Page: https://www.facebook.com/pg/DIYBFYK/shop
Website: www.beautyfromyourkitchen.com

I will appreciate any help.

Thank you,

**Comment by yassersyndicate** [comment]
its solved thank you

**Comment by Radioguy1** [comment]
Hi For some reason mine has been processing for over 24 hours 
Some products (About 1/3)  have processed and others say Processing 
I redid the settings 2 times but same result.
Sure hope you can help 
Thank you in advance 

My FB page: https://www.facebook.com/navallstoreonline
Shop Page: https://www.facebook.com/pg/navallstoreonline
Website: www.navallstore.com.com

**Comment by Radioguy1** [comment]
HI Should I Post a New Topic on this? It is still the same

**Comment by ech0ii** [comment]
Hi,
Could you also take a look on my shop page, most products are still on processing status for a day. 
https://www.facebook.com/Crypto-Swag

**Comment by ech0ii** [comment]
Hi,
If you can check please my page is at https://www.facebook.com/Crypto-Swag
its showing product is being processed for a day now. i dont know what to do.

**Comment by DonExo** [comment]
Hi, I'm also having the same issue for couple of days now.
The weird thing is that for some products it's not showing all the options available.

I'm using Shopify as a shop and integrating trough it.

My FB page is: https://www.facebook.com/mobimax.eu/
Shop page is; https://www.facebook.com/mobimax.eu/shop/
My Web site is: http://www.mobimax.eu/

**Comment by takayama10** [comment]
Hello, I'm also having the same issue. My products have been under review for a while. They are simple apparel items linked to my foundation. Please help!

Facebook: Facebook.com/Fight2Breathe

**Comment by awakenedsmiles** [comment]
Hi,
My facebook store is was processing my products and is taking too long (over 2 days) to process, and still waiting. It is mainly the clothing products. Please help.

My website address is https://awakenedsmiles.com/
My facebook page is https://www.facebook.com/awakenedsmile/?modal=admin_todo_tour

**Comment by awakenedsmiles** [comment]
Hi,
My facebook store is was processing my products and is taking too long (over 2 days) to process, and still waiting. It is mainly the clothing products. Please help.

My website address is https://awakenedsmiles.com/
My facebook page is https://www.facebook.com/awakenedsmile/?modal=admin_todo_tour

Please help, my products is still being processed.

**Comment by Ayman-attia** [comment]
Hi
it seems I have the same problem with my FB shop. I have imported my products from Woocommerce, but it is almost a week and my shop is still processing. Can you please take a look.

**Comment by Fredkalz** [comment]
Hi,
My facebook store is been on processing or review for more than 7 days now, and still waiting.  It is mainly clothing products. Please help.

My website address is https://bakazon.com/
My facebook page is https://www.facebook.com/bakazonclothing/?modal=admin_todo_tour

Please help and thanks for your feedback

**Comment by Loretta11** [comment]
I have this message on my facebook shop
Your shop is being processed
Once processing is complete, your shop will be visible to people who visit your Page. You can still create and run ads promoting your products on Facebook.
I am going crazy with this.  I am trying to set up instagram tagging. Now I have this issue. How do I get this shop to sync with Instagram so I can get the shopping item in the instagram menu and then get approved. Nothing is showing up and I've set up facebook business manager and and set up the woocommerce catalogue. Why isn't this working
https://www.facebook.com/zionstewartartist/
Instagram: zionlevystewart
PLEASE does anyone know how I can sort this.

**Comment by abdullah11099** [comment]
I'm having the same issue with my shop on facebook, below is the links to my page and online store.
www.facebook.com/sleekvalley.store/shop
www.sleekvalley.store

**Comment by vanschaikr** [comment]
Same issue here with the shop on FB
https://nummersschilderen.nl
https://www.facebook.com/Nummers-Schilderen-387217131901198/

**Comment by vanschaikr** [comment]
Same issue here with the shop on FB
https://kabeldiertje.nl/
https://www.facebook.com/pg/Kabeldiertjenl-653246731793698/shop/

**Comment by ssajith** [comment]
I initially had this issue and it was always showing processing.. I had downloaded Facebook for Woocommerce from Wordpress plugin store. Later I followed instructions from this website https://wedevs.com/99706/install-facebook-pixel-wordpress-woocommerce/ and then it was connecting properly. 

So I think if you download from Wordpress plugins, you have to configure the pixel IDs properly, else download from Facebook Ad Manager directly to get the pre configured bundle.

**Comment by vanschaikr** [comment]
What I just did is. 
- Removing the FB configuration in the WP pluging. 
- Uninstall the plugin. 
- Go to facebook to create a pixel and select Woocommerce as 3rd party. 

I uploaded the plugin, activated it and run the setup. After that everything was working. The first 2 steps were the important ones I guess as everything is now working

**Comment by gpclarke** [comment]
I'm still having this problem with one of my FB pages.  The products are active, but the IG Connect Your Instagram Business Profiles connect options on the associated business IG account is shade and I cannot select the "Not Connected" option is not available to select.  The support people only reply after 11pm PT or later...so frustrating to get this setup all week.

**Comment by Luv2flycessna** [comment]
I am having the same issue. Most of my products say your product is being processed.
